lib/puma/json.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Puma
```

Comment:
You really went above and beyond here. Better than what we had in 3.x by a mile!